Automattic/newspack-plugin
# feat(ras-acc): update otp email template

### All Submissions:






### Changes proposed in this Pull Request:

**PR 1 of 4**



This PR updates the OTP signin email with:

- A new footer per RAS-ACC email designs
- Use of primary and secondary color backgrounds
- Some alignment updates


https://github.com/Automattic/newspack-plugin/assets/17905991/1eb55e9a-802b-4a6a-a6fa-e6859682e9d4

Note: Dynamic text color and FAQ sections added in https://github.com/Automattic/newspack-plugin/pull/3033 and https://github.com/Automattic/newspack-plugin/pull/3034

Note: I am only updating the OTP email for now. Once all of the kinks are worked out it should be pretty simple to update the others.

### How to test the changes in this Pull Request:

1. Go to the WP Customizer and set a custom primary and secondary color
2. With these changes checked out, go to Newspack > Engagment > Show Advanced Settings and select the one-time password email under Transactional Email Content section:
![Screenshot 2024-04-04 at 15 05 26](https://github.com/Automattic/newspack-plugin/assets/17905991/e834ae78-c0de-40f2-acd7-7c032f6da13f)
3. Trash the post (so that we can force our updated template to be used on next otp sign in attempt)
4. As a logged out user, select the sign in link and attempt to login via email to trigger an otp email
5. Check the otp email and confirm the above points are implemented in the email design

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205668937699542
